### DISCUSSION TITLE
Public Statistics after Link Shortening

### DISCUSSION THREAD

**Discussion by Fraggle78** [YOURLS/YOURLS#3592] [Q&A] [answered]
Hello, I use the current version and the Sleeky Theme. I have set the creation of the links to public, which also works, but the link to the statistics, which is displayed, can only be viewed if you are logged in. Is it possible to make this public?
Best Greetings

**Comment by dgw** [comment, marked as answer]
https://yourls.org/docs/guide/essentials/configuration#yourls_private_infos
